components/brave_wallet_ui/components/extension/confirm-transaction-panel/index.tsx

```diff
@@ -128,6 +130,14 @@ function ConfirmTransactionPanel ({
     setShowAdvancedTransactionSettings(!showAdvancedTransactionSettings)
   }
 
+  const onClickLearnMore = () => {
+    chrome.tabs.create({ url: 'https://support.brave.com/hc/en-us/articles/5546517853325' }, () => {
```

Comment:
Needs link to actual article

Reply:
This is the actual link, article just hasn't been created yet.